resource/build-scripts/build-config-file.js

```diff
@@ -1,15 +1,29 @@
-'use strict';
-/**
- * External Dependencies
- */
-var base = require( '../../desktop-config/config-base.json' );
-var config = require( process.argv[2] );
+// TODO: evaluate if there is a more straightforward way to create config files e.g. how does calypso solve this?
+const CONFIG_SRC_DIR = '../../desktop-config/';
+const CONFIG_DEST = './desktop/config.json';
+
+const path = require( 'path' );
+const fs = require( 'fs' );
+const chalk = require( 'chalk' );
+
+const baseConfig = require( path.join( CONFIG_SRC_DIR, 'config-base.json' ) );
+
+const { NODE_ENV } = process.env;
+let envConfig = {};
+
+try {
+	envConfig = require( path.join( CONFIG_SRC_DIR, `config-${NODE_ENV}.json` ) );
```

Comment:
these should not be `NODE_ENV`

currently I see `base`, `test`, `release`, `dev`, and `updater` config files, none of which should ever be the value of `NODE_ENV`